### DISCUSSION TITLE
LN Bank - Accidentally removed all admin access to wallet

### DISCUSSION THREAD

**Discussion by BuffaloDyl** [btcpayserver/btcpayserver#5026] [Plugins & integrations]
I have created a point of sale store and an LNbank wallet on a public (not mine) BTCPay Server. I sent a few test transactions (total ~5000 sats)and was trying to create a way to share a read-only access for the wallet. On the access keys page I put in my own user name, read-only key type, and created the key.  Then of course I immediately got a 403 - Denied message, and I can no longer access the admin setting for the wallet. 

I realize this is 100% my fault and I'm happy to have donated the sats to the node owner. I would like a way to remove the wallet from my account, and maybe request an implementation that users privileges can't be overwritten by this form, maybe have to delete the key and make a new one.'

Thanks

**Comment by dennisreimann** [comment]
Thanks for the feedback, I'll fix this in LNbank v1.6 which will be released together with BTCPay Server v1.10 very soon.
